go.mod

```diff
@@ -9,23 +9,26 @@ require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-git/go-git/v5 v5.0.0
 	github.com/golang/protobuf v1.3.2
-	github.com/golangci/golangci-lint v1.23.3
+	github.com/golangci/golangci-lint v1.25.0
 	github.com/google/go-github/v29 v29.0.3
 	github.com/google/uuid v1.1.1
 	github.com/maxbrunsfeld/counterfeiter/v6 v6.2.2
 	github.com/nozzle/throttler v0.0.0-20180817012639-2ea982251481
 	github.com/olekukonko/tablewriter v0.0.4
 	github.com/pkg/errors v0.9.1
 	github.com/psampaz/go-mod-outdated v0.5.0
+	github.com/saschagrunert/go-modiff v1.0.3
 	github.com/sendgrid/rest v2.4.1+incompatible
 	github.com/sendgrid/sendgrid-go v3.5.0+incompatible
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/cobra v0.0.7
 	github.com/stretchr/testify v1.5.1
-	github.com/yuin/goldmark v1.1.25
+	github.com/yuin/goldmark v1.1.27
 	golang.org/x/net v0.0.0-20200301022130-244492dfa37a
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	google.golang.org/api v0.21.0
 	k8s.io/utils v0.0.0-20200229041039-0a110f9eb7ab
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/golangci/golangci-lint => github.com/golangci/golangci-lint v1.23.3
```

Comment:
I'm not completely happy about this workaround. Another possibility would be to move the whole go-modiff source code into this repository. On the other hand, if we would move slowly away from bazel we could simply download the golangci-lint binary in the CI without having the need to have it as direct dependency in go.mod.